src/test/encryptors/LibsodiumEncryptor.test.ts

```diff
@@ -85,3 +97,42 @@ async function setup(): Promise<{ keyProvider: FixedKeyProvider; encryptor: Libs
   const encryptor = new LibsodiumEncryptor({ keyProvider })
   return { encryptor, keyProvider }
 }
+
+/** This is a copy/pasted "legacy" version of LibsodiumEncryptor's encrypt function, which had
+ *  a bug that reused nonces across multiple encrypt operations.
+ *
+ *  It is preserved here for testing to ensure we can decrypt items encrypted with this format
+ *  */
+async function legacyEncrypt<T, U extends keyof T>(
```

Comment:
I preserved the old buggy function so we could write a unit test that proves we can still decrypt data that was encrypted using this method.